apibinding/examples/example_py_2_7.py

```
@@ -38,17 +38,17 @@ def query_until_done():
         time.sleep(1)
-        print "Job[uuid:%s] is still in processing" % job_uuid
+        print("Job[uuid:%s] is still in processing" % job_uuid)
         return query_until_done()
 
     return query_until_done()
 
 
 
 def error_if_fail(rsp):
-    success = rsp.values()[0]["success"]
+    success = list(rsp.values())[0]["success"]
```

Ruff (0.15.10)

[warning] Prefer `next(iter(rsp.values()))` over single element slice

Replace with `next(iter(rsp.values()))`

(RUF015)